src/components/dialogs/contingency-list/explicit-naming/explicit-naming-form.js

```diff
@@ -19,15 +20,60 @@ import { gridItem } from '../../../utils/dialog-utils';
 import yup from '../../../utils/yup-config';
 import { DEFAULT_ROW_VALUE } from '../contingency-list-utils';
 import ChipsArrayEditor from '../../../utils/rhf-inputs/ag-grid-table-rhf/cell-editors/chips-array-editor';
+import { ContingencyListType } from 'utils/elementType';
+
+const getSchema = (additionalTests) => {
+    let schema = yup
+        .array()
+        .of(
+            yup.object().shape({
+                [CONTINGENCY_NAME]: yup.string().nullable(),
+                [EQUIPMENT_IDS]: yup.array().of(yup.string().nullable()),
+            })
+        )
+        .test(
```

Comment:
You need to do like for filters. Try to create a contingency list by criteria, it can't works.

Reply:
Done